build/instrumentation.build.js

```diff
@@ -113,9 +111,9 @@ export function createConfig({
           include: ['**/*.js', '**/*.mjs']
         })
       ),
-      postcss(
-        mergeConfig('postcss', {
-          extract: extractCSS,
+      styles(
+        mergeConfig('styles', {
+          mode: extractCSS ? ['extract', './style.css'] : 'inject',
```

Comment:
The link to `style.css` is not added to HTML if `extractCss` is true.

Maybe we can add this plugin: https://github.com/rollup/plugins/tree/master/packages/html#readme
to do it automátically.

Reply:
I didn't add it  in this PR because It was not working before. In Gluon if I remember correctly, I had to create an HTML plugin because it was not handling well the code splitting build. I don't know if that is solved and we can using it now.